lib/lorekeeper/simple_logger.rb

```diff
@@ -25,10 +25,19 @@ class SimpleLogger < FastLogger
       UNKNOWN => COLOR_LIGHT_GRAY
     }.freeze
 
+    JSON_LOGGER_METHODS = [
+      :current_fields,
+      :state,
+      :add_thread_unsafe_fields,
+      :remove_thread_unsafe_fields,
+      :add_fields,
+      :remove_fields
+    ].freeze
+
     # \e[colorm sets a color \e[0m resets all properties
     def log_data(severity, message)
       color = SEVERITY_TO_COLOR_MAP[severity]
-      @iodevice.write("\e[#{color}m#{message}\e[0m\n")
+      @iodevice.write("\e[#{color}m#{message.gsub('\n', "\n").gsub('\t', "\t")}\e[0m\n")
```

Comment:
might be that adding rubocop-performance would advice to use `tr` instead of `gsub` but I don't remember how it behaves with escape sequeneces, YMMV.

Reply:
rubocop-performance doesn't complain about it... `'\n'` contains two characters so I guess that's why...